webapp/src/css/error-log.less

```
@@ -0,0 +1,13 @@
+.inbox-items, .targets {
+    .alert {
+      margin: 10px;
+      .error-details {
+        a {
+          float: right;
+        }
+        dt{
+          padding-top: 6px;
+        }
+      }
+    }
+}
```

I think you could simplify this (and remove the references to `.inbox-items` and `.targets`):

```suggestion
error-log > * {
  margin: 10px;
  .error-details {
    a {
      float: right;
    }
    dt{
      padding-top: 6px;
    }
  }
}
```